html/includes/print-event-short.inc.php

```diff
@@ -7,15 +7,9 @@
 }
 
 unset($icon);
-$icon_returned = geteventicon($entry['message']);
-$icon_type = $icon_returned['icon'];
-$icon_colour = $icon_returned['colour'];
+$severity_colour = eventlog_severity($entry['severity']);
 
-if ($icon_type) {
-    $icon = "<i class='fa $icon_type fa-lg' style='color:$icon_colour' aria-hidden='true'></i>";
-} else {
-    $icon = "<i class='fa fa-bookmark-o fa-lg' style='color:black' aria-hidden='true'></i>";
-}
+$icon = "<i class='fa fa-bookmark fa-lg' style='color:$severity_colour' aria-hidden='true'></i>";
```

Comment:
actually sorry to be a pain, one last change. Rather than doing style='color' can you not just create styles in styles.css called log-$colourname and then tack log-darkgrey on the end of fa-lg?